src/interface/swarm.cpp

```diff
@@ -243,8 +243,6 @@ ParArrayND<bool> Swarm::AddEmptyParticles(const int num_to_add) {
 // No particles removed: nmax_active_index unchanged
 // Particles removed: nmax_active_index is new max active index
 void Swarm::RemoveMarkedParticles() {
-  int new_max_active_index = -1; // TODO(BRR) this is a magic number, needed for Defrag()
```

Comment:
@brryan Want to comment on this?

Reply:
Sorry I missed this @pgrete, thanks for the ping @AndrewGaspar. I don't think this will break anything unless we store particles in negative array indices in the future. It's just a little messy so I added the TODO to go back at some point and think through the logic again. If the `TODO(BRR)` is registering on some list of issues, I'm fine with just making the comment `// This is a magic number, needed for Defrag()` if that's OK.

Reply:
The variable isn't being used.

Reply:
Hm ok this may just be vestigial code then. Let's say it's fine to go ahead and delete this now, but I'll take a closer look at this in the near future to ensure the logic is sound (the unit test gives me some confidence).